packages/jupyter-ai/src/components/code-blocks/copy-button.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useCallback, useRef } from 'react';
+
+import { copyIcon } from '@jupyterlab/ui-components';
+
+import { TooltippedIconButton } from '../mui-extras/tooltipped-icon-button';
+
+enum CopyStatus {
+  None,
+  Copying,
+  Copied
+}
+
+const COPYBTN_TEXT_BY_STATUS: Record<CopyStatus, string> = {
+  [CopyStatus.None]: 'Copy to Clipboard',
```

Comment:
This casing is inconsistent with the other commands in the mini-toolbar, whose titles are in sentence case.

```suggestion
  [CopyStatus.None]: 'Copy to clipboard',
```

Reply:
Done in latest revision. 👍